doc/docusaurus/docs/reference/plutus-tx-compiler-options.md

```diff
@@ -2,6 +2,11 @@
 sidebar_position: 5
```

Comment:
This appears if you look at the rendered markdown on GhiHub. Is it needed to produce the [fancy documentation](https://plutus.cardano.intersectmbo.org/docs/reference/plutus-tx-compiler-options)?

![Sidebar](https://github.com/user-attachments/assets/cc0c3078-291e-4427-939e-f8553e37fa14)

Reply:
Yes